data/cncf.yaml

```diff
@@ -2811,6 +2855,8 @@
       url: https://github.com/carvel-dev/carvel
       check_sets:
         - community
+        - code
+        - docs
```

Comment:
The `docs` check set is a subset of the `code` check set, so it isn't needed in this case. Please remove it.